### ISSUE TITLE
about roi_img

### ISSUE THREAD

**Issue by niannianmeng** [cleardusk/3DDFA#42]
Hi, jianzhu,

I found that if i do not use dlib_landmarks,  but use anothor rectangles. the 3DDFA result is not so good. 
 is it that roi_img containing  face is not sufficient?
what additional requirements should roi_img meet?

thank you for you reply!

**Comment by niannianmeng** [comment]
Hi, jianzhu,
thanks for you reply! I still have some questions about the training process:
is the bounding box calculated using landmarks equivalent to the "ground truth box" in readme.md ?
is  the landmarks mentioned above the 3D landmarks or 2D landmarks(such as dlib)?
will the train loss function combined with wpdc and vdc be released?

**Comment by cleardusk** [comment]
The training details will be updated in the next two months.
